pkgs/top-level/all-packages.nix

```diff
@@ -30252,9 +30252,6 @@ with pkgs;
 
   ecs-agent = callPackage ../applications/virtualization/ecs-agent { };
 
-  inherit (recurseIntoAttrs (callPackage ../applications/editors/ed { }))
-    ed edUnstable;
```

Comment:
`edUnstable` should go to aliases as `throw`.

Reply:
Done.